waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/DatabaseMappingService.java

```diff
@@ -37,6 +37,11 @@
    */
   DatabaseMapping databaseMapping(@NotNull String databaseName) throws NoSuchObjectException;
 
+  void checkTable(String databaseName, String tableName,
+      DatabaseMapping mapping) throws NoSuchObjectException;
+
+  List<String> filterTables(String databaseName, List<String> tableNames, DatabaseMapping mapping);
```

Comment:
This could do with some javadoc explaining what the intention is, I got a bit confused reading the implementations below. Is this returning a list of table names that should be filtered out? Or a list of table names that don't pass the filter? What are the criteria for the "filtering" operation?

Reply:
yeah... I'll add a javadoc 👍